hash_reader_test.go

```diff
@@ -87,7 +87,7 @@ func TestConsumeRemainderNext(t *testing.T) {
 }
 
 func TestIonReaderContract(t *testing.T) {
-	file, err := ioutil.ReadFile("ion_hash_tests.ion")
+	file, err := ioutil.ReadFile("ion-hash-test/ion_hash_tests.ion")
```

Comment:
I think now that everything is passing we do not need a test file locally and should use the original file (I'm not sure if this change on this line is for that).
In case we are still reading the tests from a locally placed file, this PR might be helpful: https://github.com/amzn/ion-hash-dotnet/pull/42

Reply:
Yes `ion-hash-test/ion_hash_tests.ion` is the original file from the git submodule.
Note that this PR removes the temporary `ion_hash_tests.ion` since all the tests are passing now.